git_machete/git_operations.py

```diff
@@ -123,6 +123,10 @@ def of(value: str) -> Optional["FullCommitHash"]:
                                 'Consider posting an issue on https://github.com/VirtusLab/git-machete/issues/new')
         return None
 
+    @staticmethod
+    def is_valid(value: str) -> bool:
+        return value is not None and len(value) == 40
```

Comment:
What if the value has length 40, but has characters outside `[0-9a-f]`? pls cover this case by tests